projects/addon-doc/src/services/theme.options.ts

```diff
@@ -0,0 +1,14 @@
+import {InjectionToken} from '@angular/core';
+
+export const TUI_THEME_STORAGE_DEFAULT_KEY = `tuiTheme` as const;
+
+export const TUI_THEME_STORAGE_KEY = new InjectionToken<string>(
+    `[TUI_THEME_STORAGE_KEY]`,
+    {factory: () => TUI_THEME_STORAGE_DEFAULT_KEY},
+);
+
+export const TUI_THEME_DEFAULT_NAME = `Open-source` as const;
```

Comment:
`Taiga UI` maybe? I hope someday we can bundle multiple themes like `Taiga UI 2024`  :)

Reply:
fixed